Dashboard/va.gov.artemis.ui/Views/Outcome/AddEdit.cshtml

```diff
@@ -26,7 +26,7 @@
         <div class="col-md-6">
             <table class="table table-bordered table-db table-horizontal">
                 <tbody>
-                    <tr class="info delivery-outcome">
```

Comment:
Is this unused? Does it introduce CSS Style regression or any jQuery?

Reply:
This class was removed to force the field to show on all the types of pregnancies, not only on the ones marked with the delivery-outcome class.

Reply:
The class was added by mistake initially.